tests/e2e/map/map.spec.js

```diff
@@ -1629,7 +1629,7 @@ test.describe('Map interactions', () => {
         })
       })
 
-      await page.route(/colormaps\/GHRSST_L4_MUR_Sea_Ice_Concentration/, async (route) => {
+      await page.route(/colormaps\/GHRSST_Sea_Surface_Temperature/, async (route) => {
```

Comment:
Is there a reason this changed?

Reply:
Yes, GHRSST_L4_MUR_Sea_Ice_Concentration is no longer a valid colormap. It was changed to be  GHRSST_L4_MUR_Sea_Ice_Concentration  -> GHRSST_Sea_Ice_Concentration